src/mesh/Draco_Mesh_Builder.t.hh

```diff
@@ -166,7 +177,8 @@ Draco_Mesh_Builder<FRT>::build_mesh(rtt_mesh_element::Geometry geometry) {
 
   std::shared_ptr<Draco_Mesh> mesh(new Draco_Mesh(
       dimension, geometry, cell_type, cell_to_node_linkage, side_set_flag,
-      side_node_count, side_to_node_linkage, coordinates, global_node_number));
+      side_node_count, side_to_node_linkage, coordinates, global_node_number,
+      {}, {}, {}, {}, face_type));
```

Comment:
I'm not crazy about the empty brackets here.  It's not hard to look up the argument list as defined by the constructor, but maybe we could use inline named argument declarations here.   What do you think? Is it worth the hassle?

Reply:
Are keyword arguments possible? Apologies for my confusion... Some variables have been initialized as empty and used here, with consistent naming.

Reply:
I'm not sure about it for this UseCase.  Have you tried `{} --> vector<int> foobar{}` or a similar signature?

Reply:
It didn't appear to work...  `vector<int>()` works, which is somewhat more descriptive.  What about this?